### PR TITLE
remove carriers

### LINKED ISSUE #292: Remove "Carrier"
The ST department is going to buy an actual cell phone for dispatchers to text passengers manually _not_ using email.

### PR DESCRIPTION
**Pull request by Anbranin**
closes #292

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -28,21 +23,21 @@
     t.string "phone"
   end
 
-  create_table "log_entries", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```
Comment (Anbranin) [inline db/schema.rb:31]: why

File: spec/system/passenger_show_spec.rb [hunk 2]
```diff
@@ -14,18 +13,16 @@
     context 'viewing passenger' do
       let(:number) { @passenger.phone.delete('-') }
       context 'passenger is subscribed to sms' do
-        it 'displays a SMS address ' do
-          @passenger.update(subscribed_to_sms: true, carrier: @carrier)
+        it 'say so' do
```
Comment (jforgue) [inline spec/system/passenger_show_spec.rb:16]: Maybe a more descriptive test name would be good here lol.

File: spec/system/passenger_show_spec.rb [hunk 3]
```diff
@@ -14,18 +13,16 @@
...
+          @passenger.update(subscribed_to_sms: true)
           click_on 'View'
           expect(page).to have_text 'Subscribed to SMS: Yes'
-          expect(page).to have_text "SMS address: #{number}#{@carrier.gateway_address}"
         end
       end
       context 'passenger is not subscribed to sms' do
-        it 'does not display a SMS address' do
+        it 'says so' do
```
Comment (jforgue) [inline spec/system/passenger_show_spec.rb:23]: Same here.

**Review by jforgue — CHANGES_REQUESTED** [review]
Looks good but you missed the routes file.
https://github.com/umts/st-pax/blob/a89feafcc7ca891af723bf26f4baacfc66391e56/config/routes.rb#L12

File: spec/system/passenger_show_spec.rb (hunk 2 above)
Comment (Anbranin) [reply spec/system/passenger_show_spec.rb:16]: First of all, this isn't a change, I just moved stuff. Second of all, it's supposed to be read in conjunction with the context. Third of all, I don't care, so I changed it.

**Review by jforgue — APPROVED** [review]
Looks good

**Pull request merged**